circularSeekBar/src/main/java/me/tankery/lib/circularseekbar/CircularSeekBar.kt

```diff
@@ -545,9 +575,10 @@ class CircularSeekBar @JvmOverloads constructor(
             R.styleable.cs_CircularSeekBar_cs_circle_y_radius,
             DEFAULT_CIRCLE_Y_RADIUS
         )
+
         pointerStrokeWidth = attrArray.getDimension(
             R.styleable.cs_CircularSeekBar_cs_pointer_stroke_width,
-            DEFAULT_POINTER_STROKE_WIDTH
+            DEFAULT_POINTER_WIDTH
```

Comment:
Wondering why we change this? `DEFAULT_POINTER_STROKE_WIDTH` sounds more unified with `pointerStrokeWidth`